src/style.css

```
@@ -1 +1,43 @@
 /* write your styles here */
+body {
+  margin: 0;
+}
+
+.important,
+.interesting,
+.surprising,
+#header,
+#footer {
+  width: 300px;
+  height: 100px;
+  margin: 0 auto;
+  transition: 0.3s;
```

таким способом ты задаешь transition на все свойства, так делать не стоит
задавай только на то что анимируешь
```suggestion
  transition: height 0.3s;
```